test/joinTables.js

```diff
@@ -373,4 +373,71 @@ describe('JoinTables', function() {
     });
   });
 
+
+  describe('junction table between the same model', function() {
+    var collections = {};
+
+    before(function() {
+
+      collections.foo = {
+        tableName: 'foo',
+        connection: 'foo',
+        migrate: 'safe',
+        attributes: {
+          id: {
+            type: 'integer',
+            autoIncrement: true,
+            primaryKey: true,
+            unique: true
+          }
+        },
+        follows: {
+          collection: 'foo',
+          through: 'bar',
+          via: 'from'
```

Comment:
Is this line and line 402 supposed to be the same?  Shouldn't one of the `via` values be `'to'`?